src/panels/config/integrations/integration-panels/zwave_js/dialog-zwave_js-add-node.ts

```diff
@@ -623,15 +638,13 @@ class DialogZWaveJSAddNode extends LitElement {
 
   private _startInclusion(
     qrProvisioningInformation?: QRProvisioningInformation,
-    qrCodeString?: string,
-    plannedProvisioningEntry?: PlannedProvisioningEntry
+    dsk?: string
   ): void {
     if (!this.hass) {
       return;
     }
     this._lowSecurity = false;
-    const specificDevice =
-      qrProvisioningInformation || qrCodeString || plannedProvisioningEntry;
+    const specificDevice = qrProvisioningInformation || dsk;
```

Comment:
why are we removing these other inclusion options? I believe those are still valid

Reply:
but unused 🤷‍♂️